src/apis/UserApi.ts

```diff
@@ -37,6 +36,16 @@ export class UserApi extends AbstractApi {
     userId: string,
     params: UserParams = {}
   ): Promise<User> {
+    // The Clerk server API requires metadata fields to be stringified
+
+    if (params.publicMetadata && !(typeof params.publicMetadata == 'string')) {
```

Comment:
❓ Why do you check if the public metadata is already string? To cover for cases where the user has already stringify it?

Reply:
Yes, we have at least one known case of a dev passing it stringified.

Reply:
👍